nemo/collections/asr/losses/rnnt.py

```diff
@@ -87,24 +187,21 @@ def __init__(self, num_classes, reduction='mean_batch'):
 
             reduction: Type of reduction to perform on loss. Possibly values are `mean`, `sum` or None.
                 None will return a torch vector comprising the individual loss values of the batch.
+
+            loss_name: String that is resolved into an RNNT loss function. Available list of losses
+                is ininitialized in `RNNT_LOSS_RESOLVER` dictionary.
+
+            loss_kwargs: Optional Dict of (str, value) pairs that are passed to the instantiated loss
+                function.
         """
         super(RNNTLoss, self).__init__()
 
-        if not WARP_RNNT_AVAILABLE:
-            raise ImportError(
-                "Could not import `warprnnt_pytorch`.\n"
-                "Please visit https://github.com/HawkAaron/warp-transducer "
-                "and follow the steps in the readme to build and install the "
-                "pytorch bindings for RNNT Loss, or use the provided docker "
-                "container that supports RNN-T loss."
-            )
-
         if reduction not in [None, 'mean', 'sum', 'mean_batch']:
             raise ValueError('`reduction` must be one of [mean, sum, mean_batch]')
```

Comment:
none is missing from the list.

Reply:
This should not be the str "none" - thats only special for the RNNT losses above. This one should be proper None.